lib/mayaUsd/render/vp2ShaderFragments/UsdPreviewSurfaceLightAPI3.xml

```diff
@@ -0,0 +1,107 @@
+<!--
+========================================================================
+Copyright 2018 Pixar
+Copyright 2022 Autodesk
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+========================================================================
+-->
+<fragment_graph name="UsdPreviewSurface" ref="UsdPreviewSurface" class="FragmentGraph" version="1.0" feature_level="0">
+    <fragments>
+        <!-- Fragments for extracting computed dot products. -->
+        <fragment_ref name="NwFaceCameraIfNAN" ref="NwFaceCameraIfNAN"/>
+
+        <!-- Fragments for computing material. -->
+        <fragment_ref name="opacityToTransparency" ref="opacityToTransparency"/>
+        <fragment_ref name="usdPreviewSurfaceLightingAPI3" ref="usdPreviewSurfaceLightingAPI3"/>
```

Comment:
Only change compared to V2 is to use API3 suffix everywhere. See other XML file for flat shading change.